PIMbench/cpp-knn/baselines/GPU/Makefile

```diff
@@ -0,0 +1,11 @@
+# Compiler
+CXX := nvcc
+
+# Executable name
+EXEC := knn.out
+
+all:
+	$(CXX) -o $(EXEC) -I./inc -I/sw/ubuntu-22.04/cuda/extras/CUPTI/include -L/sw/ubuntu-22.04/cuda/extras/CUPTI/lib64 -DONLY_TIME_KERNELS test.cpp -I./inc knncuda.cu -lcuda -lcublas -lcudart -Wno-deprecated-gpu-targets
```

Comment:
Can you change it to be the same as vector add cuda makefile

Reply:
Somehow this made my execution slower when I made the change but maybe it was something with portal